<!doctype html>

<html class="no-js" lang="en">

<head>


	<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

	George Tiley

	Journal Theme by https://jekyllthemes.io
	Premium + free Jekyll themes for your blog or website.

	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->


	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">

	<!-- Page Info -->
	<link rel="shortcut icon" href="/images/flavicon.png">
	<title>Mutation Rate Variation – George Tiley</title>
	<meta name="description" content="<h3 id="variation-of-rates-and-spectra-between-and-within-species">Variation of rates and spectra between and within species</h3>
">

	<!-- Twitter Card -->
	<meta name="twitter:card" content="summary_large_image">
	<meta name="twitter:title" content="Mutation Rate Variation – George Tiley">
	<meta name="twitter:description" content="<h3 id="variation-of-rates-and-spectra-between-and-within-species">Variation of rates and spectra between and within species</h3>
">
	<meta name="twitter:image:src" content="http://localhost:4000/images/staticContent/mutation-resized.png">

	<!-- Facebook OpenGraph -->
	<meta property="og:title" content="Mutation Rate Variation – George Tiley" />
	<meta property="og:description" content="<h3 id="variation-of-rates-and-spectra-between-and-within-species">Variation of rates and spectra between and within species</h3>
" />
	<meta property="og:image" content="http://localhost:4000/images/staticContent/mutation-resized.png" />

	
	<!-- Font Embed Code -->
	<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400|Muli:400,400i,600" rel="stylesheet">
	

	<!-- Styles -->
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="/css/style.css">
	
	<!-- Icons -->
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/solid.js" integrity="sha384-GXi56ipjsBwAe6v5X4xSrVNXGOmpdJYZEEh/0/GqJ3JTHsfDsF8v0YQvZCJYAiGu" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/brands.js" integrity="sha384-0inRy4HkP0hJ038ZyfQ4vLl+F4POKbqnaUB6ewmU4dWP0ki8Q27A0VFiVRIpscvL" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/fontawesome.js" integrity="sha384-NY6PHjYLP2f+gL3uaVfqUZImmw71ArL9+Roi9o+I4+RBqArA2CfW1sJ1wkABFfPe" crossorigin="anonymous"></script>

	
	<!-- Custom Styles -->
	<style></style>
	

	
	<!-- Analytics Code -->
	
	

	
	<!-- Extra Header JS Code -->
	
	
	
</head>


<body class="loading ajax-loading" data-site-url="http://localhost:4000" data-page-url="/project/mutation">


	<header class="header">

	<div class="header-image header-image--on" style="background-image: url(/images/staticContent/mutation-resized.png);"></div>
	<div class="header-image"></div>

	<div class="header-overlay"></div>

	<div class="header__content">

		
		<a href="/" class="header__title">
			George Tiley
		</a>
		

		<p class="header__tagline">Computational Molecular Evolution</p>

		<div class="menu">
			<div class="menu__toggle js-menu-toggle">
				<div class="menu__toggle__icon"><span></span></div>
			</div>
			<div class="menu__wrap">
				<ul class="menu__list">
					
					<li class="menu__list__item">
						<a href="/" class="menu__list__item__link">Blog</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/projects/" class="menu__list__item__link">Projects</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/madgrass" class="menu__list__item__link">MADGRASS</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/publications" class="menu__list__item__link">Publications</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/software" class="menu__list__item__link">Software</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/teaching" class="menu__list__item__link">Teaching</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/contact-plain" class="menu__list__item__link">Contact</a>
					</li>
					
				</ul>
				<ul class="socials">
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	<li class="socials__item">
		<a href="https://github.com/gtiley" target="_blank" class="socials__item__link" title="Github">
			<i class="fab fa-github" aria-hidden="true"></i>
		</a>
	</li>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
</ul>
			</div>
		</div>

	</div>

</header>


	<div class="loader"><svg width="120" height="30" viewBox="0 0 120 30" xmlns="http://www.w3.org/2000/svg"><circle cx="15" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="60" cy="15" r="9" fill-opacity="0.3"><animate attributeName="r" from="9" to="9" begin="0s" dur="0.8s" values="9;15;9" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="0.5" to="0.5" begin="0s" dur="0.8s" values=".5;1;.5" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="105" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle></svg></div>

	<div class="page-loader"></div>

	
	<div class="page">

		<div class="page__content" data-page-title="Mutation Rate Variation – George Tiley" data-image="/images/staticContent/mutation-resized.png">

			<section class="intro">

	<div class="wrap">

		<h1>Mutation Rate Variation</h1>
		<p>estimating spontaneous mutation rates from whole-genome sequences</p>

	</div>

</section>

<section class="single">

	<div class="wrap">

		<h3 id="variation-of-rates-and-spectra-between-and-within-species">Variation of rates and spectra between and within species</h3>

<p>Very recently it has become feasible to characterize the mutations that arise in a single generation by deep sequencing of whole genomes from pedigrees. When parents and their offspring are available, one can simply count the mutations and what type of mutations are in the offspring but absent from the parents. There are a number of caveats though, reviewed in Yoder and Tiley (2021) <sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote" rel="footnote">1</a></sup>.<br /></p>

<p>Measuring <em>de novo</em> mutation rates can yield quite surprising patterns that invoke the need for deeper understandings of lineage-specific differences in DNA repair mechansism, as revealed by our investigation in mouse lemurs <sup id="fnref:2" role="doc-noteref"><a href="#fn:2" class="footnote" rel="footnote">2</a></sup>. For example, mouse lemurs have a reduced number of C-&gt;T transitions at CpG sites.</p>

<p><img src="/images/projects/2020-08-01-mutation/Figure2.png" /></p>

<p><strong>Figure 2 from Campbell, Tiley et al. 2021 <sup id="fnref:2:1" role="doc-noteref"><a href="#fn:2" class="footnote" rel="footnote">2</a></sup></strong> Most primates have a majority of their C-&gt;T transitions in CpG sites, whereas in mouse lemurs, these are the minority (A). While unexpected, the pedigree-based results are supported by phylogenetic relaxed-clock models that partition sites by substitution type. Mouse lemurs show a near-equal rate of C-&gt;T transitions at CpG and non-CpG sites (B), while the C-&gt;T transition rate at CpG sites is higher in other sampled primates.<br /></p>

<p>I am do not have active research in this area at the moment, but some nice people continue to include me in the conversation <sup id="fnref:3" role="doc-noteref"><a href="#fn:3" class="footnote" rel="footnote">3</a></sup>. I am to pivot from examining species-specific differences to evaluating ecological correlates with mutation rates with population-level data in the near future.<br /></p>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1" role="doc-endnote">
      <p>Yoder AD, <strong>Tiley GP</strong>. 2021. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.16007">The challenge and promise of estimating the <em>de novo</em> mutation rate from whole-genome comparisons among closely related individuals.</a> <em>Molecular Ecology</em> 30:6087-6100. <a href="#fnref:1" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:2" role="doc-endnote">
      <p>Campbell CR<sup>†</sup>, <strong>Tiley GP<sup>†</sup></strong>, Poelstra JW, Hunnicutt KE, Larsen PA, Lee H-J, Thorne JL, dos Reis M, Yoder AD. 2021. <a href="https://www.nature.com/articles/s41437-021-00446-5">Pedigree-based and phylogenetic methods support surprising patterns of mutation rate and spectrum in the gray mouse lemur.</a> <em>Heredity</em> 127:233-244. <a href="#fnref:2" class="reversefootnote" role="doc-backlink">&#8617;</a> <a href="#fnref:2:1" class="reversefootnote" role="doc-backlink">&#8617;<sup>2</sup></a></p>
    </li>
    <li id="fn:3" role="doc-endnote">
      <p>Bergeron LA, Besenbacher S, Turner T, Versoza CJ, Wang RJ, Price AL, Armstrong E, Riera M, Carlson J, Chen H-Y, Hahn MW, Harris K, Jleppe AS, López-Nandam EH, Moorjani P, Pfeifer SP, <strong>Tiley GP</strong>, Yoder AD, Zhang G, Schierup MH. 2022. <a href="https://elifesciences.org/articles/73577">The mutationathon highlights the importance of reaching standardization in the estimates of pedigree-based germline mutation rates.</a> <em>eLife</em> 11:e73577. <a href="#fnref:3" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
  </ol>
</div>


	</div>

</section>

		</div>

	</div>


	<footer class="footer">

	<div class="footer__copyright">
		<span>© 2022 George Tiley</span>
		<a href="https://jekyllthemes.io" target="_blank">Jekyll Themes</a>
	</div>

</footer>


	<!-- Javascript Assets -->
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src="/js/journal-min.js"></script>

	
	<!-- Extra Footer JS Code -->
	
	


</body>

</html>