<!doctype html>

<html class="no-js" lang="en">

<head>


	<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

	George Tiley

	Journal Theme by https://jekyllthemes.io
	Premium + free Jekyll themes for your blog or website.

	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->


	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">

	<!-- Page Info -->
	<link rel="shortcut icon" href="/images/flavicon.png">
	<title>Polyploidy – George Tiley</title>
	<meta name="description" content="<h3 id="diversification-key-innovation-and-the-grand-challenges">Diversification, key innovation, and the grand challenges</h3>
">

	<!-- Twitter Card -->
	<meta name="twitter:card" content="summary_large_image">
	<meta name="twitter:title" content="Polyploidy – George Tiley">
	<meta name="twitter:description" content="<h3 id="diversification-key-innovation-and-the-grand-challenges">Diversification, key innovation, and the grand challenges</h3>
">
	<meta name="twitter:image:src" content="http://localhost:4000/images/staticContent/grassdist-resized.png">

	<!-- Facebook OpenGraph -->
	<meta property="og:title" content="Polyploidy – George Tiley" />
	<meta property="og:description" content="<h3 id="diversification-key-innovation-and-the-grand-challenges">Diversification, key innovation, and the grand challenges</h3>
" />
	<meta property="og:image" content="http://localhost:4000/images/staticContent/grassdist-resized.png" />

	
	<!-- Font Embed Code -->
	<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400|Muli:400,400i,600" rel="stylesheet">
	

	<!-- Styles -->
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="/css/style.css">
	
	<!-- Icons -->
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/solid.js" integrity="sha384-GXi56ipjsBwAe6v5X4xSrVNXGOmpdJYZEEh/0/GqJ3JTHsfDsF8v0YQvZCJYAiGu" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/brands.js" integrity="sha384-0inRy4HkP0hJ038ZyfQ4vLl+F4POKbqnaUB6ewmU4dWP0ki8Q27A0VFiVRIpscvL" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/fontawesome.js" integrity="sha384-NY6PHjYLP2f+gL3uaVfqUZImmw71ArL9+Roi9o+I4+RBqArA2CfW1sJ1wkABFfPe" crossorigin="anonymous"></script>

	
	<!-- Custom Styles -->
	<style></style>
	

	
	<!-- Analytics Code -->
	
	

	
	<!-- Extra Header JS Code -->
	
	
	
</head>


<body class="loading ajax-loading" data-site-url="http://localhost:4000" data-page-url="/project/polyploidy">


	<header class="header">

	<div class="header-image header-image--on" style="background-image: url(/images/staticContent/grassdist-resized.png);"></div>
	<div class="header-image"></div>

	<div class="header-overlay"></div>

	<div class="header__content">

		
		<a href="/" class="header__title">
			George Tiley
		</a>
		

		<p class="header__tagline">Molecular Evolution</p>

		<div class="menu">
			<div class="menu__toggle js-menu-toggle">
				<div class="menu__toggle__icon"><span></span></div>
			</div>
			<div class="menu__wrap">
				<ul class="menu__list">
					
					<li class="menu__list__item">
						<a href="/" class="menu__list__item__link">Blog</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/projects/" class="menu__list__item__link">Projects</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/madgrass" class="menu__list__item__link">MADGRASS</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/publications" class="menu__list__item__link">Publications</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/software" class="menu__list__item__link">Software</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/teaching" class="menu__list__item__link">Teaching</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/contact-plain" class="menu__list__item__link">Contact</a>
					</li>
					
				</ul>
				<ul class="socials">
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	<li class="socials__item">
		<a href="https://github.com/gtiley" target="_blank" class="socials__item__link" title="Github">
			<i class="fab fa-github" aria-hidden="true"></i>
		</a>
	</li>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
</ul>
			</div>
		</div>

	</div>

</header>


	<div class="loader"><svg width="120" height="30" viewBox="0 0 120 30" xmlns="http://www.w3.org/2000/svg"><circle cx="15" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="60" cy="15" r="9" fill-opacity="0.3"><animate attributeName="r" from="9" to="9" begin="0s" dur="0.8s" values="9;15;9" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="0.5" to="0.5" begin="0s" dur="0.8s" values=".5;1;.5" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="105" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle></svg></div>

	<div class="page-loader"></div>

	
	<div class="page">

		<div class="page__content" data-page-title="Polyploidy – George Tiley" data-image="/images/staticContent/grassdist-resized.png">

			<section class="intro">

	<div class="wrap">

		<h1>Polyploidy</h1>
		<p>inferring the presence of polyploidy and its importance in plants</p>

	</div>

</section>

<section class="single">

	<div class="wrap">

		<h3 id="diversification-key-innovation-and-the-grand-challenges">Diversification, key innovation, and the grand challenges</h3>

<p>Polyploidy was long appreciated in plants, but took on a new importance since the advent of genomics as it simultaneously became a source of novelty and a bioinformatic challenge. I have worked on methods for detecting ancient whole-genome duplications (WGDs) with distributions of evolutionary distances between paralogs within a species <sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote" rel="footnote">1</a></sup> or distributions of gene count data <sup id="fnref:2" role="doc-noteref"><a href="#fn:2" class="footnote" rel="footnote">2</a></sup> or gene trees <sup id="fnref:3" role="doc-noteref"><a href="#fn:3" class="footnote" rel="footnote">3</a></sup> to place WGDs in a phylogenetic context. I am broadly interested in the consequences of polyploidy on genome architecture and gene family evolution.<br /></p>

<p>I am currently investigating how phasing data in polyploids can be useful for population genomics and inferring phylogenetic networks. This includes developing methods and pipelines for phasing <sup id="fnref:4" role="doc-noteref"><a href="#fn:4" class="footnote" rel="footnote">4</a></sup>, and how to apply those phased data to demographic analyses under the coalescent. Polyploids were usually tossed from population genetic studies or dealt with in a very . My goal is to get everybody to the haploid level so we can work with mixed ploidy systems under a unified coalescent framework. We are just scratching the surface of developing and applying these methods, but simulations suggest there can be large benefits to phasing on network estimation with allopolyploids.<br /></p>

<p><img src="/images/projects/2020-08-02-polyploidy/D.png" />
<strong>Figure 2 from Tiley et al. 2021 <sup id="fnref:4:1" role="doc-noteref"><a href="#fn:4" class="footnote" rel="footnote">4</a></sup></strong> Simulation results for whether a network is correct given different treatments of the sequence data as phased as unphased. Either having all of the phased haplotypes (phased) or simply having one phased haplotype sampled per locus (pick one) seems to do well across all simulation conditions with 40 or more loci. A consensus approach also works well once enough loci are sampled. Treating allopolyploid data as an unphased diploid (genotype) can be problematic even as the number of loci increase though.<br /></p>

<p>The utility of genotyping and phasing polyploids is not yet clear, at least to me. Will correctly genotyping individuals at various ploidy levels lead to better estimates of site frequency spectra and ultimately more accurate model inferences? I am working on these areas now as part of <a href="/madgrass">MADGRASS</a>.<br /></p>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1" role="doc-endnote">
      <p><strong>Tiley GP</strong>, Barker MS, Burleigh JG. 2018. <a href="https://academic.oup.com/gbe/article/10/11/2882/5100828">Assessing the Performance of <em>Ks</em> Plots for Detecting Ancient Whole Genome Duplications.</a> <em>Genome Biology and Evolution</em> 10:2882-2898. <a href="#fnref:1" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:2" role="doc-endnote">
      <p><strong>Tiley GP</strong>, Ané C, Burleigh JG. 2016. <a href="https://academic.oup.com/gbe/article/8/4/1023/2574077">Evaluating and characterizing ancient whole-genome duplications in plants with gene count data.</a> <em>Genome Biology and Evolution</em> 8:1023-1037. <a href="#fnref:2" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:3" role="doc-endnote">
      <p>Li Z<sup>†</sup>,<strong>Tiley GP<sup>†</sup></strong>, Galuska SR, Reardon CR, Kidder TI, Rundell RJ, Barker MS. 2018. <a href="https://www.pnas.org/content/115/18/4713.short">Multiple large-scale gene and genome duplications during the evolution of hexapods.</a> <em>PNAS</em> 115:4713-4718. <a href="#fnref:3" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:4" role="doc-endnote">
      <p><strong>Tiley GP</strong>, Crowl AA, Manos PS, Sessa EB, Solís-Lemus C, Yoder AD, Burleigh JG. 2021. <a href="https://www.biorxiv.org/content/10.1101/2021.05.04.442457v1.abstract">Phasing alleles improves network inference with allopolyploids</a> <em>bioRxiv</em> doi: https://doi.org/10.1101/2021.05.04.442457<br /> <a href="#fnref:4" class="reversefootnote" role="doc-backlink">&#8617;</a> <a href="#fnref:4:1" class="reversefootnote" role="doc-backlink">&#8617;<sup>2</sup></a></p>
    </li>
  </ol>
</div>


	</div>

</section>

		</div>

	</div>


	<footer class="footer">

	<div class="footer__copyright">
		<span>© 2023 George Tiley</span>
		<a href="https://jekyllthemes.io" target="_blank">Jekyll Themes</a>
	</div>

</footer>


	<!-- Javascript Assets -->
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src="/js/journal-min.js"></script>

	
	<!-- Extra Footer JS Code -->
	
	


</body>

</html>