<!doctype html>

<html class="no-js" lang="en">

<head>


	<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

	George Tiley

	Journal Theme by https://jekyllthemes.io
	Premium + free Jekyll themes for your blog or website.

	- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->


	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">

	<!-- Page Info -->
	<link rel="shortcut icon" href="/images/flavicon.png">
	<title>Phased Haplotype Sequences from Polyploids – George Tiley</title>
	<meta name="description" content="A new pipeline for phasing sequence data of any ploidy">

	<!-- Twitter Card -->
	<meta name="twitter:card" content="summary_large_image">
	<meta name="twitter:title" content="Phased Haplotype Sequences from Polyploids – George Tiley">
	<meta name="twitter:description" content="A new pipeline for phasing sequence data of any ploidy">
	<meta name="twitter:image:src" content="http://localhost:4000/images/blogPosts/2021-06-12-Phasing-Polyploids/2021-06-12_Fig8.png">

	<!-- Facebook OpenGraph -->
	<meta property="og:title" content="Phased Haplotype Sequences from Polyploids – George Tiley" />
	<meta property="og:description" content="A new pipeline for phasing sequence data of any ploidy" />
	<meta property="og:image" content="http://localhost:4000/images/blogPosts/2021-06-12-Phasing-Polyploids/2021-06-12_Fig8.png" />

	
	<!-- Font Embed Code -->
	<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400|Muli:400,400i,600" rel="stylesheet">
	

	<!-- Styles -->
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="/css/style.css">
	
	<!-- Icons -->
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/solid.js" integrity="sha384-GXi56ipjsBwAe6v5X4xSrVNXGOmpdJYZEEh/0/GqJ3JTHsfDsF8v0YQvZCJYAiGu" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/brands.js" integrity="sha384-0inRy4HkP0hJ038ZyfQ4vLl+F4POKbqnaUB6ewmU4dWP0ki8Q27A0VFiVRIpscvL" crossorigin="anonymous"></script>
	<script defer src="https://use.fontawesome.com/releases/v5.1.1/js/fontawesome.js" integrity="sha384-NY6PHjYLP2f+gL3uaVfqUZImmw71ArL9+Roi9o+I4+RBqArA2CfW1sJ1wkABFfPe" crossorigin="anonymous"></script>

	
	<!-- Custom Styles -->
	<style></style>
	

	
	<!-- Analytics Code -->
	
	

	
	<!-- Extra Header JS Code -->
	
	
	
</head>


<body class="loading ajax-loading" data-site-url="http://localhost:4000" data-page-url="/blog/phasing-polyploids">


	<header class="header">

	<div class="header-image header-image--on" style="background-image: url(/images/blogPosts/2021-06-12-Phasing-Polyploids/2021-06-12_Fig8.png);"></div>
	<div class="header-image"></div>

	<div class="header-overlay"></div>

	<div class="header__content">

		
		<a href="/" class="header__title">
			George Tiley
		</a>
		

		<p class="header__tagline">Molecular Evolution</p>

		<div class="menu">
			<div class="menu__toggle js-menu-toggle">
				<div class="menu__toggle__icon"><span></span></div>
			</div>
			<div class="menu__wrap">
				<ul class="menu__list">
					
					<li class="menu__list__item">
						<a href="/" class="menu__list__item__link">Blog</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/projects/" class="menu__list__item__link">Projects</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/madgrass" class="menu__list__item__link">MADGRASS</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/publications" class="menu__list__item__link">Publications</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/software" class="menu__list__item__link">Software</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/teaching" class="menu__list__item__link">Teaching</a>
					</li>
					
					<li class="menu__list__item">
						<a href="/contact-plain" class="menu__list__item__link">Contact</a>
					</li>
					
				</ul>
				<ul class="socials">
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	<li class="socials__item">
		<a href="https://github.com/gtiley" target="_blank" class="socials__item__link" title="Github">
			<i class="fab fa-github" aria-hidden="true"></i>
		</a>
	</li>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	
</ul>
			</div>
		</div>

	</div>

</header>


	<div class="loader"><svg width="120" height="30" viewBox="0 0 120 30" xmlns="http://www.w3.org/2000/svg"><circle cx="15" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="60" cy="15" r="9" fill-opacity="0.3"><animate attributeName="r" from="9" to="9" begin="0s" dur="0.8s" values="9;15;9" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="0.5" to="0.5" begin="0s" dur="0.8s" values=".5;1;.5" calcMode="linear" repeatCount="indefinite" /></circle><circle cx="105" cy="15" r="15"><animate attributeName="r" from="15" to="15" begin="0s" dur="0.8s" values="15;9;15" calcMode="linear" repeatCount="indefinite" /><animate attributeName="fill-opacity" from="1" to="1" begin="0s" dur="0.8s" values="1;.5;1" calcMode="linear" repeatCount="indefinite" /></circle></svg></div>

	<div class="page-loader"></div>

	
	<div class="page">

		<div class="page__content" data-page-title="Phased Haplotype Sequences from Polyploids – George Tiley" data-image="/images/blogPosts/2021-06-12-Phasing-Polyploids/2021-06-12_Fig8.png">

			<section class="intro">

	<div class="wrap">

		<h1>Phased Haplotype Sequences from Polyploids</h1>
		<p>12 June 2021</p>

	</div>

</section>

<section class="single">

	<div class="wrap">

		<h3 id="why-should-data-be-phased">Why should data be phased?</h3>

<p>I originally became interested in phasing sequence data for polyploids when I was learning about genotyping for accurate estimation of site frequency spectra for various grass species in Madagascar. At the same time, I was applying the MSci model <sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote" rel="footnote">1</a></sup> for divergence time estimation with introgression for the mouse lemur work, and it seemed like phasing data would be a good strategy for enabling easier analyses of polyploids with network models. I was recently involved in some generation and analysis of target enrichment data <sup id="fnref:2" role="doc-noteref"><a href="#fn:2" class="footnote" rel="footnote">2</a></sup>, so that seemed like a good place to start seeing as these data are arguably more accessible than other types of genomic data.<br />
Honestly, I needed to convince myself that phasing actually mattered for improving analyses of polyploids, specifically allopolyploids. There were a few empirical examples that seemed important <sup id="fnref:3" role="doc-noteref"><a href="#fn:3" class="footnote" rel="footnote">3</a></sup> <sup id="fnref:4" role="doc-noteref"><a href="#fn:4" class="footnote" rel="footnote">4</a></sup>, but the methods were not easily scalable for many individuals. So I first started with some simulations under a simple scenario with a single allopolyploid (Fig. 1). Simulations were done under the MSC with very simple assumptions about rate variation, pop sizes, and the substitution model.<br /></p>

<p><img src="/images/blogPosts/2021-06-12-Phasing-Polyploids/Fig2.png" /></p>

<p><strong>Fig. 1 — Species network used for simulation.</strong> The divergence times in expected substitutions per site are given for each node, and h is the hybrid node where two alleles enter from both t and v. E is an allotetraploid while other species are diploid. Nucleotide divergence was reduced by dividing all τ by 10 or increased by multiplying all τ by 10. ILS was increased by halving the distance between τ_h and τ_u and between τ_u and τ_s either once (for the low ILS condition) or twice (for the moderate ILS condition).<br /></p>

<p>Networks were then estimated using a topology-based pseudolikelihood approach with PhyloNetworks <sup id="fnref:5" role="doc-noteref"><a href="#fn:5" class="footnote" rel="footnote">5</a></sup>. The results were motivating enough that phasing should be useful for investigating at least allopolyploids. Five different input data sets were prepared for the analysis:<br /></p>
<ol>
  <li>True - the known simulated gene trees</li>
  <li>Phased - gene trees estimated from the true haplotype data</li>
  <li>Genotype - gene trees estimated from sequences where variant sites (within an individual) are collapsed into their IUPAC codes. This would be the result of using some genoytping software without phasing.</li>
  <li>Consensus - gene trees based on sequences where only one base per variant site is used.</li>
  <li>Pick One - gene trees estimated with the tye haplotype data, but only 1 allele is randomly selected per individual.
The phased data greatly outperformed haploype consensus data and genotype data for accurately estimating a network (Fig. 2). It is also important to note that a decent number of loci are needed for network estimation. I remember starting out in molecular evolution by cloning Sanger loci in polyploids. Cloning is a lot of work and I have seen other approached for obtaining lower numbers of long phased loci advocated for, but I am not convinced this is the best way forward given sampling error - especially for high ILS scenarios.<br /></li>
</ol>

<p><img src="/images/blogPosts/2021-06-12-Phasing-Polyploids/Fig4.png" /></p>

<p><strong>Fig. 2 — Proportion of simulations that correctly identify the allopolyploid lineage.</strong> The x-axes are the number of loci sampled for each simulation. Th y-axes are the proportions of correct networks. Results are based on networks estimated with a single reticulation, even if that network was considered less optimal than networks with zero or two reticulations. We saved the true gene trees from the simulations, while we estimated gene trees with the phased and unphased (Genotype, Consensus, and Pick One) data.<br /></p>

<h3 id="a-fast-and-scalable-phasing-strategy">A fast and scalable phasing strategy</h3>

<p>Along with my coauthor Andy Crowl, we developed a pipeline (Fig. 3) called PATÉ (Phased Alleles from Target Enrichment data). The core of PATÉ is H-PoPG <sup id="fnref:6" role="doc-noteref"><a href="#fn:6" class="footnote" rel="footnote">6</a></sup>. Currently, <em>de novo</em> haplotype consensus assemblies are used at a reference for each individual. Reads are then realigned to loci for genotyping and those genotypes go through some filtering. H-PoPG is then used to phase those variants passing filters, and phased fasta files are output for analysis. I think this is a good strategy for analyses of multiple species where there is no single reference genome, but some power is lost compared to joint genotyping if you have a reference genome and are more interested in population genomics.<br /></p>

<p><img src="/images/blogPosts/2021-06-12-Phasing-Polyploids/Fig3.png" /></p>

<p><strong>Fig. 3 — PATÉ Phasing Pipeline.</strong> Overview of data input, output, and steps taken to phase alleles. Input data are assumed to be paired-end Illumina reads and reference sequences for each individual are required (consensus loci from HybPiper can be used). The ploidy of each individual must be specified. Only biallelic sites are used for phasing.</p>

<p>What I like most about PATÉ is that were are able to be rigorous about genotype quality while also being fast. And because we operate on individuals, the genotyping and phasing can be distributed easily on a cluster to scale to many individuals as is typical of contemporary phylogenomics projects.<br /></p>

<p>We have analyzed several empirical data sets to-date using PATÉ, but the one from the <a href="https://www.biorxiv.org/content/10.1101/2021.05.04.442457v1">preprint</a> are <em>Dryopteris</em> ferns <sup id="fnref:7" role="doc-noteref"><a href="#fn:7" class="footnote" rel="footnote">7</a></sup>. We chose this group because the parents for the allopolyploids are assumed known and because the system is actually quite complex. We do a number of analyses covered in the preprint that suggest the phased data is important for network estimation, especially when there are multiple introgression events involved. Some good news is that when scenarios are relatively simple (i.e. two parents and their allopolyploid), both phased and the haplotype consensus data can do well (Fig. 4). Perhaps alarmingly though, the haplotype consensus data can get the direction of introgression wrong at an alarming rate when there are too few loci.<br /></p>

<p><img src="/images/blogPosts/2021-06-12-Phasing-Polyploids/Fig8.png" /></p>

<p><strong>Fig. 4 — Network search results for three-taxon Dryopteris example.</strong> Both phased and unphased haplotype consensus data recovered the same network topology with introgression occurring in the expected direction. The major topology is indicated by the blue edge and the minor edge (direction of introgression) is shown in tan. The inheritance probability γ was slightly higher in the haplotype consensus data (γunphased = 0.455). Bar plots show the proportion of 100 replicates when sampling four or 40 loci that correctly detect one reticulation based on the pseudolikelihood scores (top), correctly estimate the network with introgression going from one of the diploids into D. campyloptera (middle), and estimate a network where the direction of introgression is from D. campyloptera into one of the diploids (bottom).<br /></p>

<p>There might be scenarios where phasing is a good option. I think the haplotype consensus data should be fine for analyzing really ancient events. I advocate that the phasing might be beneficial for tackling more recent species complexes though.<br /></p>

<h3 id="tutorial">Tutorial</h3>

<p>Currently, I suggest reading the <a href="https://www.biorxiv.org/content/10.1101/2021.05.04.442457v1">preprint</a> and following the instructions on <a href="https://github.com/gtiley/Phasing">GitHub</a>. A more complete tutorial is in the works but should be up in the coming months.<br /></p>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1" role="doc-endnote">
      <p>Flouri T, Jiao X, Rannala B, Yang Z. 2020. A Bayesian Implementation of the Multispecies Coalescent Model with Introgression for Phylogenomic Analysis. <strong>Mol. Biol. Evol.</strong> 37:1211-1223. <a href="#fnref:1" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:2" role="doc-endnote">
      <p>Breinholt JW, Carey SB, Tiley GP, Davis EC, Endara L, McDaniel SF, Neves LG, Sessa EB, et al. 2021. A target enrichment probe set for resolving the flagellate plant tree of life. <strong>Appl. Plant Sci.</strong> 9:e11406. <a href="#fnref:2" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:3" role="doc-endnote">
      <p>Oberprieler C, Wagner F, Tomasello S, Konowalik K. 2017. A permutation approach for inferring species networks from gene trees in polyploid complexes by minimising deep coalescences. <strong>Methods in Ecology and Evolution</strong> 8:835-849. <a href="#fnref:3" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:4" role="doc-endnote">
      <p>Eriksson JS, de Sousa F, Bertrand YJK, Antonelli A, Oxelman B, Pfeil BE. 2018. Allele phasing is critical to revealing a shared allopolyploid origin of Medicago arborea and M. strasseri (Fabaceae). <strong>BMC Evol. Biol.</strong> 18:9. <a href="#fnref:4" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:5" role="doc-endnote">
      <p>Solis-Lemus C, Bastide P, Ané C. 2017. PhyloNetworks: A Package for Phylogenetic Networks. <strong>Mol. Biol. Evol.</strong> 34:3292-3298. <a href="#fnref:5" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:6" role="doc-endnote">
      <p>Xie M, Wu Q, Wang J, Jiang T 2016. H-PoP and H-PoPG: heuristic partitioning algorithms for single individual haplotyping of polyploids. Bioinformatics 32:3735-3744. <a href="#fnref:6" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:7" role="doc-endnote">
      <p>Sessa EB, Zimmer EA, Givnish TJ. 2012. Unraveling reticulate evolution in North American Dryopteris (Dryopteridaceae). BMC. Evol. Biol. 12:104. <a href="#fnref:7" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
  </ol>
</div>


	</div>

</section>

		</div>

	</div>


	<footer class="footer">

	<div class="footer__copyright">
		<span>© 2022 George Tiley</span>
		<a href="https://jekyllthemes.io" target="_blank">Jekyll Themes</a>
	</div>

</footer>


	<!-- Javascript Assets -->
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src="/js/journal-min.js"></script>

	
	<!-- Extra Footer JS Code -->
	
	


</body>

</html>